src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingToAndFromManyAddressesSpecification_Steps.cs

```diff
@@ -0,0 +1,198 @@
+﻿using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
+using Stratis.Bitcoin.IntegrationTests.TestFramework;
+using Xunit.Abstractions;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    using Stratis.Bitcoin.Features.Consensus;
+    using Stratis.Bitcoin.Features.Wallet;
+    using NBitcoin;
+    using System.Collections.Generic;
+    using System.Linq;
+    using Stratis.Bitcoin.Features.Wallet.Controllers;
+    using Stratis.Bitcoin.Features.Wallet.Models;
+    using System;
+    using FluentAssertions;
+
+    public partial class SendingToAndFromManyAddressesSpecification : BddSpecification
+    {
+        private NodeBuilder nodeBuilder;
+        private const string WalletName = "mywallet";
+        private const string WalletPassword = "123456";
+        private const string WalletAccountName = "account 0";
+        private CoreNode nodeOne;
+        private CoreNode nodeTwo;
+        private long stratisReceiverTotalAmount;
+        private Key keySender;
+        private TransactionBuildContext transactionBuildContext;
+        private int CoinBaseMaturity;
+
+        // Output helper allows test step names to be logged.
+        public SendingToAndFromManyAddressesSpecification(ITestOutputHelper outputHelper) : base(outputHelper) { }
+
+        protected override void BeforeTest()
+        {
+            this.nodeBuilder = NodeBuilder.Create();
+        }
+
+        protected override void AfterTest()
+        {
+            this.nodeBuilder.Dispose();
+        }
+
+        private void funds_in_a_single_address_on_node1_wallet()
+        {
+            this.nodeOne = this.nodeBuilder.CreateStratisPowNode();
+            this.nodeTwo = this.nodeBuilder.CreateStratisPowNode();
+
+            this.nodeBuilder.StartAll();
+            this.nodeOne.NotInIBD();
+            this.nodeTwo.NotInIBD();
+
+            this.nodeOne.FullNode.WalletManager().CreateWallet(WalletPassword, WalletName);
+            this.nodeTwo.FullNode.WalletManager().CreateWallet(WalletPassword, WalletName);
```

Comment:
All of us appear to be repeating this bit of code everywhere, perhaps something like a NodeNetworkBuilder  could be useful (perhaps when writing next test between us)